Odata-docs/client/v7/query-options.md

```diff
@@ -57,75 +104,75 @@ var people = context.People.IncludeCount();
 var people = context.People.IncludeCount(true);
 ```
 
-For `GET https://host/service/EntitySet?$count=false`:
+For `GET https://host/service/People?$count=false`:
 
 ``` csharp
 var people = context.People.IncludeCount(false);
 ```
 
 ## $orderby
 
-For `GET https://host/service/EntitySet?$orderby=Prop`:
+For `GET https://host/service/People?$orderby=FirstName`:
 
 ``` csharp
 var people = context.People.OrderBy(c => c.FirstName);
 ```
 
-For `GET https://host/service/EntitySet?$orderby=Prop desc`:
+For `GET https://host/service/People?$orderby=FirstName desc`:
 
 ``` csharp
 var people = context.People.OrderByDescending(c => c.FirstName);
 ```
 
-For `GET https://host/service/EntitySet?$orderby=PropCol/$count`:
+For `GET https://host/service/People?$orderby=Trips/$count`:
 
 ``` csharp
 var people = context.People.OrderBy(c => c.Trips.Count);
 ```
 
 ## $skip
 
-For `GET https://host/service/EntitySet?$skip=3:
+For `GET https://host/service/People?$skip=3`:
 
 ``` csharp
 var people = context.People.Skip(3);
 ```
 
 ## $top
 
-For `GET https://host/service/EntitySet?$top=3:
+For `GET https://host/service/People?$top=3`:
 
 ``` csharp
 var people = context.People.Take(3);
 ```
 
 ## $expand
 
-For `GET https://host/service/EntitySet?$expand=Trips:
+For `GET https://host/service/People?$expand=Trips`:
 
 ``` csharp
 var people = context.People.Expand(c => c.Trips);
 ```
 
 ## $select
 
-For `GET https://host/service/EntitySet // selects all columns by default
+For `GET https://host/service/People` // selects all columns by default
 OR
```

Comment:
```suggestion

OR

```
Add newline spacing because the 2 lines above and below the "OR" seem to be rendering on one line

Reply:
#Resolved